xml/gl.xml

```diff
@@ -9140,6 +9141,7 @@ typedef unsigned int GLhandleARB;
             <param><ptype>GLenum</ptype> <name>target</name></param>
             <param><ptype>GLuint</ptype> <name>index</name></param>
             <param><ptype>GLuint</ptype> <name>buffer</name></param>
+            <glx type="render" opcode="356"/>
```

Comment:
Something seems to have gone wrong with the XML portion of the change BeginTransformFeedback has the right opcode, but at least a handful of the other ones I spot-checked don't from my reading of our internal driver code generation data and the specs I dug up on the old internal Khronos bugzilla.  Can you double-check these?

Reply:
OK, confirmed after some offline back-and-forth with @nwnk that this is me reading github webdiffs wrong.  After fetching & looking at the diff locally with more context, the opcodes look right.